runtime/compiletime/memory_instructions.c

```diff
@@ -1,78 +1,48 @@
 #include <assert.h>
 
-#include "types.h"
+#include "wasm_module_instance.h"
 
-uint32_t
+INLINE uint32_t
 instruction_memory_size()
 {
-	return local_sandbox_context_cache.memory.size / WASM_PAGE_SIZE;
+	return wasm_memory_get_page_count(current_wasm_module_instance.memory);
 }
 
 // All of these are pretty generic
 INLINE float
 get_f32(uint32_t offset)
 {
-	assert(offset + sizeof(float) <= local_sandbox_context_cache.memory.size);
-
-	char *mem_as_chars = (char *)local_sandbox_context_cache.memory.start;
-	void *address      = &mem_as_chars[offset];
-
-	return *(float *)address;
+	return wasm_memory_get_f32(current_wasm_module_instance.memory, offset);
```

Comment:
These functions are all symbols that back invocations that are code-gened by aWsm. This means for example that if we wanted to namespace `get_f32`, we would have to change aWsm, SLEdge, all other runtimes, and recompile all modules.